tests/kernel/threads/dynamic_thread_stack/testcase.yaml

```diff
@@ -3,7 +3,7 @@ common:
   min_ram: 32
   integration_platforms:
     - qemu_x86
```

Comment:
@tejlmand should these be updated also since they have variants and the underlying board.yml has the full name, or is it okay to just keep the shortened name here?

Reply:
twister currently takes `qemu_x86` as `qemu_x86/ia32` and other variants are named explicitly, aren't they ?
```
CONFIG_BOARD="qemu_x86"                                                                                                                                                                                                                                               CONFIG_BOARD_REVISION=""                                                                                                                                                                                                                                              CONFIG_BOARD_QEMU_X86=y                                                                                                                                                                                                                                               CONFIG_BOARD_QEMU_X86_IA32=y      
```